### PR TITLE
feat: Make landing page mobile responsive

### PR DESCRIPTION
**Pull request by Maniktherana**
partially addresses #500 

Adds mobile-friendly footer and navbar and makes the landing page more mobile responsive. Also removes the functionality of the navbar hiding on scroll down temporarily.
This is mostly patchwork but its a start

https://github.com/vitessio/arewefastyet/assets/14011425/476865cb-b06e-46d5-a69b-4ceac28f4901

### CODE REVIEW

File: website/src/pages/HomePage/components/HeroMobile.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Link } from "react-router-dom";
```
Comment (frouioui) [inline website/src/pages/HomePage/components/HeroMobile.jsx:2]: License header is missing here :)

**Review by frouioui — APPROVED** [review]
One review feedback otherwise it looks great to me !

File: website/src/pages/HomePage/components/HeroMobile.jsx (hunk 1 above)
Comment (Maniktherana) [reply website/src/pages/HomePage/components/HeroMobile.jsx:18]: Done

**Comment by frouioui** [comment]
Thank you for this contribution!

**Pull request merged**